External/FEXCore/include/FEXCore/Utils/DeferredSignalMutex.h

```diff
@@ -165,19 +229,44 @@ namespace FEXCore {
     &FEXCore::ForkableSharedMutex::lock,
     &FEXCore::ForkableSharedMutex::unlock>;
 
+  class ScopedSignalMasker final {
+    public:
+      ScopedSignalMasker() = default;
+
+      void Mask(uint64_t Mask) {
+#ifndef _WIN32
+        // Mask all signals, storing the original incoming mask
+        ::syscall(SYS_rt_sigprocmask, SIG_SETMASK, &Mask, &OriginalMask, sizeof(OriginalMask));
+#endif
+      }
+
+      // Move-only type
+      ScopedSignalMasker(const ScopedSignalMasker&) = delete;
+      ScopedSignalMasker& operator=(ScopedSignalMasker&) = delete;
+      ScopedSignalMasker(ScopedSignalMasker &&rhs) = default;
```

Comment:
this should also have the move assignment operator defaulted

Reply:
Oops, you're right. Fixed.